internal/resource/nvml-device.go

```diff
@@ -99,3 +99,24 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) GetFabricIds() (string, string, error) {
+	gfInfo, ret := d.GetGpuFabricInfo()
+	if ret != nvml.SUCCESS {
+		return "", "", ret
+	}
+
+	// ClusterUUID
+	byteSlice := gfInfo.ClusterUuid[:]
+	clusterUUID := fmt.Sprintf("%08x-%04x-%04x-%04x-%012x",
+		byteSlice[0:4],
+		byteSlice[4:6],
+		byteSlice[6:8],
+		byteSlice[8:10],
+		byteSlice[10:16])
+
+	// CliqueID
+	cliqueId := fmt.Sprintf("%d", uint64(gfInfo.CliqueId))
```

Comment:
Since `CliqueId` is already a `uint32` should we not just be able to `Sprintf` it directly?
```suggestion
	cliqueId := fmt.Sprintf("%d", gfInfo.CliqueId)
```